build/yamls/antrea-windows-with-ovs.yml

```diff
@@ -40,51 +40,31 @@ data:
     $ErrorActionPreference = "Stop"
     $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
     $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/')
-    $env:PATH = $env:PATH + ";$mountPath/Windows/System32;$mountPath/k/antrea/bin;$mountPath/openvswitch/usr/bin;$mountPath/openvswitch/usr/sbin"
+    $env:PATH = $env:PATH + ";$mountPath/k/antrea/bin;$mountPath/openvswitch/usr/bin;$mountPath/openvswitch/usr/sbin"
     & antrea-agent --config=$mountPath/etc/antrea/antrea-agent.conf --logtostderr=false --log_dir=c:/var/log/antrea --alsologtostderr --log_file_max_size=100 --log_file_max_num=4 --v=0
   Install-OVSDriver.ps1: |
     $ErrorActionPreference = "Stop"
     $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
     $mountPath = ($mountPath.Replace('\', '/')).TrimEnd('/')
-    $OVSDriverDir = "$mountPath\openvswitch\driver"
-
-    # Check if OVSExt driver is already installed
-    $driverStatus = netcfg -q ovsext
-    if ($driverStatus -like '*not installed*') {
-      # Install OVS Driver
-      $result = netcfg -l $OVSDriverDir/ovsext.inf -c s -i OVSExt
-      if ($result -like '*failed*') {
-        Write-Host "Failed to install OVSExt driver: $result"
-        exit 1
-      }
-      Write-Host "OVSExt driver has been installed"
+    $OVSInstallScript = "$mountPath\k\antrea\Install-OVS.ps1"
+    if (-not (Test-Path $OVSInstallScript)) {
+      Write-Host "Installation script not found: $OVSInstallScript, you may be using an invalid antrea-windows container image"
+      exit 1
     }
-
-    # Check if the VC redistributable is already installed.
-    $OVSRedistDir="$mountPath\openvswitch\redist"
-    if (Test-Path $OVSRedistDir) {
-      $dllFound = $false
-      $paths = $env:PATH -split ';'
-      foreach ($path in $paths) {
-        $dllFiles = Get-ChildItem -Path $path -Filter "vcruntime*.dll" -File -ErrorAction SilentlyContinue
-        if ($dllFiles.Count -gt 0) {
-          $dllFound = $true
-          break
-        }
-      }
-
-      # vcruntime dlls are not installed on the host, then install the binaries.
-      if (-not $dllFound) {
-        Get-ChildItem $OVSRedistDir -Filter *.exe | ForEach-Object {
-          Start-Process -FilePath $_.FullName -Args '/install /passive /norestart' -Verb RunAs -Wait
-        }
-      }
+    $installOVSJob = Start-Job -FilePath $OVSInstallScript -ArgumentList "","","$mountPath/openvswitch",$false,""
+    $installOVSJob | Receive-Job -Wait
```

Comment:
I think this is just a lack of knowledge on my side, but if we start a job and wait for its completion, how is that different from invoking `$OVSInstallScript` directly?

Reply:
@rajnkamr Please explain the rationale for using a job here. Since we do not want the initContainer to exit until the script completes, it seems that starting a Windows job is just adding complexity?

Reply:
wrt https://github.com/antrea-io/antrea/pull/6383#discussion_r1630759515, Start-Job provides an option to get more job related debugging information and when the job finishes, can use the receive-Job cmdlet to get the results of the job failure and success and better manage script lifecycle.
Currently since we do not need to run this asynchronously in background so  using  "$installOVSJob | Receive-Job -Wait" is allowing to wait for script to finish in foreground .

Reply:
@rajnkamr could you give some details or examples about "more job related debugging information" which are necessary but not provided if we directly run the powershell script without a job?

Reply:
@wenyingd ,Idea was to store/show the information specific to driver wrt signed vs unsigned, there are various ways to do it and store the information for user for example, $jobOutput = $installOVSJob | Receive-Job -Wait and then save to a file $jobOutput | Out-File "C:\Output.txt" , 
Also we could start installation of ovs drivers asynchronously without blocking the current execution with wait ?
when we optimize ovs installation and provide more granularity wrt parallel run, I am open to do it later with the start-job approach if it looks complex !

Reply:
If the advantage is to save the installation output to a file, we already provide the LOG capability in script Install-OVS.ps1 (https://github.com/antrea-io/antrea/blob/main/hack/windows/Install-OVS.ps1#L72) which supports writing the installation to both stdout and file `install_ovs.log` which located in the path where Install-OVS.ps1 exists. 

If we switch to use job, some problems are introduced for the logic to identify where the script file locates because script file is translated as a script block if called inside from a job and we can't use the original logic to get the file directory, so I add some code in this change to use [$env:Temp](https://github.com/antrea-io/antrea/pull/6383/files#diff-757171652cab6408022bc6f9d62d928162b44bce8bbaf676ba4d8e8c6c432fe9) in the job scenario to avoid exceptions.

For container installation case, I don't think a persistence on the installation details is better than stdout which can be read from the kubelet logs. If we expect to persis the information, we may also need to think about mounting volumns for the log file (more changes are needed, e.g., parameter for log path?), which seems not worthy.

@antoninbas @XinShuYang any thoughts?

Reply:
> Also we could start installation of ovs drivers asynchronously without blocking the current execution with wait

I don't think we can do that. At least we don't want to exit from the initContainer until everything is ready and the antrea-agent can start.

@wenyingd I would recommend running the script normally in the foreground, with logs going to both stdout and a (persisted) file on the host filesystem if possible. Based on your comment, it looks like it may already be what we are doing.
It's not worth adding complexity and workarounds to the script so that it can be run as a job IMO.